ui/src/main.rs

```diff
@@ -155,6 +159,8 @@ enum LogProfile {
     Debug,
     /// print everything including tracing logs to the terminal
     Trace,
+    /// like trace but include warp logs
+    Trace2,
```

Comment:
idk, it's all documented via the `--help` option and without running that command, no one would know about the different trace options anyway. i had the same thoughts and considered something like "supertrace". but felt that it would be more work and less convenient, in exchange for minimal gain